include/MiniLua/stdlib.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef STDLIB_HPP
+#define STDLIB_HPP
```

Comment:
```suggestion
#ifndef MINILUA_STDLIB_HPP
#define MINILUA_STDLIB_HPP
```

to prevent potential conflicts